src/maybe/maybe.py

```diff
@@ -19,9 +21,17 @@
     from typing_extensions import ParamSpec, TypeAlias, TypeGuard
 
 
+try:
+    import result  # pyright: ignore[reportMissingImports]
+
```

Comment:
No need for this extra blank line here?

Reply:
True, Python doesn't care, but most code formatters do. I can remove it if you prefer, but if you would like to incorporate something like `ruff`, it will want that blank line.

Reply:
Okay, let's leave it for now